GCC/RP2350_ARM_NTZ/non_secure/port.c

```diff
@@ -867,7 +867,7 @@ static void prvTaskExitError( void )
      * should instead call vTaskDelete( NULL ). Artificially force an assert()
      * to be triggered if configASSERT() is defined, then stop here so
      * application writers can catch the error. */
-    configASSERT( portGET_CRITICAL_NESTING_COUNT() == ~0UL );
+    configASSERT( portGET_CRITICAL_NESTING_COUNT( 0 ) == ~0UL );
```

Comment:
nit : `portGET_CORE_ID()` is probably more suitable here.
```suggestion
    configASSERT( portGET_CRITICAL_NESTING_COUNT( portGET_CORE_ID() ) == ~0UL );
```